src/animated/Controller.ts

```diff
@@ -325,7 +325,7 @@ export class Controller<State extends Indexable = any> {
 
   // Update the props and animations
   private _run(props: UpdateProps<State>, onEnd: OnEnd) {
-    if (globalConfig) props = { ...props, ...globalConfig }
+    if (globalConfig) props.config = { ...props.config, ...globalConfig }
```

Comment:
This needs to account for `props.config` being a function, so we'll need to move this somewhere better.

Reply:
right, so _run isn't the proper place. how about diff, where it unpacks config? or does it affect multiple places?

Reply:
I made it so `globalConfig` is merged in `_animate`, which means it doesn't get processed by `_diff`, but I think I'm okay with that.